components/schema-migrator/main.go

```diff
@@ -96,8 +129,58 @@ func main() {
 	}
 
 	if err != nil {
-		fmt.Printf("Error during migration, %s\n", err)
+		return fmt.Errorf("during migration: %s", err)
+	}
+
+	return nil
+}
+
+func quitCloudSqlProxy() error {
+	fmt.Println("# QUITTING CLOUD SQL PROXY #")
+	matches, err := filepath.Glob("/proc/*/exe")
```

Comment:
Is this generic enough (to cover all platforms used by us)?
Could we consider github.com/mitchellh/go-ps?